### PR TITLE
Pin `min` capacity to `desired`  capacity when rolling back

### PR DESCRIPTION
**Pull request by ajordens**
image

### CODE REVIEW

**Comment by ajordens** [comment]
relates to spinnaker/orca#1290

File: app/scripts/modules/amazon/serverGroup/details/rollback/rollbackServerGroup.html [hunk 1]
```diff
@@ -42,13 +42,16 @@
...
-              <strong>min</strong>: {{serverGroup.capacity.min}},
+              <strong>min</strong>: {{serverGroup.capacity.desired}},
               <strong>max</strong>: {{ serverGroup.capacity.max }},
               <strong>desired</strong>: {{ serverGroup.capacity.desired }}
-            ]</li>
+            ]<br/>(minimum capacity pinned at {{serverGroup.capacity.desired}} to prevent autoscaling down during rollback)
+            </li>
             <li>Disable {{ serverGroup.name }}</li>
+            <li>Restore minimum capacity of <em>{{ command.rollbackContext.restoreServerGroupName || 'previous server group' }}</em> [
+              <strong>min</strong>: 1
```
Comment (anotherchrisberry) [inline app/scripts/modules/amazon/serverGroup/details/rollback/rollbackServerGroup.html]: did you mean to hardcode `1` here?

**Comment by ajordens** [comment]
Hah no. That was me generating a screenshot. Oops.

**Comment by ajordens** [comment]
Should be much more improved now.

**Comment by ajordens** [comment]
I'll merge and release this after `orca` goes out

**Pull request merged**